portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,55 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>Welcome to Jason's Portfolio</h1>
     </div>
+
+    <div id="header">
+      <img class="circular--square" src="../Images/headshot.png" alt="My headshot">
+      <p> Jason Ah Chuen <br>
+          Stanford University | Class of 2022 <br>
+          B.S. Candidate | Computer Science <br>
+          jahchuen (at) stanford (dot) edu
+      </p>
+    </div>
+
+    <div id="content1">
+      <h3> I am a sophomore student at Stanford University, passionate about CS + social good, 
```

Comment:
Any particular reason you've chosen to use h1/h3/h5 instead of h1/h2/h3?  Could you achieve your desired result by styling h2 and h3 to your taste?

A word about my review style: I generally try not make the same comment multiple times, esp. if the comment is about a "guiding principle" sort of thing. My hope is that you'll apply the principle throughout the PR as appropriate.